cloud/aws-functions/orphaned_openshift_instances.py

```diff
@@ -0,0 +1,59 @@
+# Remove openshift resources.
+import logging
+import datetime
+import boto3
+import sys
+from time import sleep
+from botocore.exceptions import ClientError
+from boto3.exceptions import Boto3Error
+
+
+def isInstanceToTerminate(instance):
```

Comment:
is there some reason to use camel case here and snake case for other functions naming?